core/src/main/java/com/exadel/aem/toolkit/bundle/lists/util/ListsHelper.java

```diff
@@ -53,13 +53,13 @@
     }
     
     /**
-     * Retrieves the list of item resources from {@code listPagePath} and adapts them to {@link GenericItem} model
+     * Retrieves the list of item resources from {@code listPagePath} and adapts them to {@link SimpleListItem} model
      * @param resourceResolver an instance of ResourceResolver
```

Comment:
Please start params definitions with capital case letter as elsewhere and recheck other similar places

Reply:
fixed in all list-related files